input/community/speakers/christian-weyer.md

```diff
@@ -0,0 +1,27 @@
+Title: Christian Weyer
+Location: Neustadt am Main, Germany
+Email: christian.weyer@thinktecture.com
+Language:
+  - English
+Topics:
+  - .NET
+  - ASP.NET
+  - ASP.NET Web API
+  - Architecture
+  - Azure
+  - Blazor
+  - C#
+  - JavaScript
+  - Microservices
+  - Security
+  - Serverless
+  - SignalR
+  - Visual Studio
+  - Visual Studio Code
+  - Web Development
+Blog: https://www.thinktecture.com/de/christian-weyer
+Twitter: @christianweyer
```

Comment:
FYI - it's expecting a full URL here, so https://twitter.com/christianweyer

Reply:
Thx!